libosinfo: 1.9.0 -> 1.10.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/libosinfo/versions.


meta.description for libosinfo is: "GObject based library API for managing information about operating systems, hypervisors and the (virtual) hardware devices they can support"

meta.homepage for libosinfo is: "https://libosinfo.org/"

meta.changelog for libosinfo is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- Warning: no invocation of /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0/bin/osinfo-install-script had a zero exit code or showed the expected version
- Warning: no invocation of /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0/bin/osinfo-detect had a zero exit code or showed the expected version
- Warning: no invocation of /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0/bin/osinfo-query had a zero exit code or showed the expected version
- 0 of 3 passed binary check by having a zero exit code.
- 0 of 3 passed binary check by having the new version present in output.
- found 1.10.0 with grep in /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0
- found 1.10.0 in filename of file in /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
20 total rebuild path(s)

19 package rebuild(s)

19 x86_64-linux rebuild(s)
1 x86_64-darwin rebuild(s)


First fifty rebuilds by attrpath
dropbox-cli
gnome-photos
gnome.file-roller
gnome.gnome-books
gnome.gnome-boxes
gnome.gnome-control-center
gnome.gnome-documents
gnome.gnome-terminal
gnome.gnome-user-share
gnome.nautilus
gnome.nautilus-python
kgx
libosinfo
nautilus-open-any-terminal
pantheon.file-roller
pantheon.file-roller-contract
phosh
tracker-miners
virt-manager
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libosinfo https://github.com/r-ryantm/nixpkgs/archive/024128ed43469fcd95c46fe7dfc03a8f937ebe4f.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0
ls -la /nix/store/sn387qwssgz1q690x7r842wilrlrdxcg-libosinfo-1.10.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>gnome.gnome-documents</li>
  </ul>
</details>
<details>
  <summary>18 packages built:</summary>
  <ul>
    <li>dropbox-cli</li>
    <li>gnome-photos</li>
    <li>gnome.file-roller</li>
    <li>gnome.gnome-books</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-user-share</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>kgx</li>
    <li>libosinfo</li>
    <li>nautilus-open-any-terminal</li>
    <li>pantheon.file-roller</li>
    <li>pantheon.file-roller-contract</li>
    <li>phosh</li>
    <li>tracker-miners</li>
    <li>virt-manager</li>
  </ul>
</details>

---

###### Maintainer pings

cc @bjornfor for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

jonringer commented:
@GrahamcOfBorg eval

jtojnar reviewed (approved):
https://gitlab.com/libosinfo/libosinfo/-/blob/v1.10.0/NEWS

Changes look okay, we do not want to switch to libsoup3 for now to avoid symbol conflicts.

jtojnar committed:
osinfo-db-tools: 1.9.0 → 1.10.0

https://gitlab.com/libosinfo/osinfo-db-tools/-/blob/v1.10.0/NEWS

- Switch to libsoup3
- Add meta.changelog
- Format the expression with nixpgks-fmt

jtojnar committed:
libosinfo: 1.9.0 → 1.10.0

https://gitlab.com/libosinfo/libosinfo/-/blob/v1.10.0/NEWS

- Add meta.changelog

jtojnar committed:
osinfo-db: 20211216 → 20220214

https://gitlab.com/libosinfo/osinfo-db/-/commits/v20220214
